server/server.js

```diff
@@ -114,6 +120,47 @@ app.get(`${apiBaseUrl}/participants`,
     return res.json(result);
   }));
 
+app.get(`${apiBaseUrl}/participant`,
+  keycloak.allowRolesMiddleware('health_authority', 'ministry_of_health', 'employer'),
+  keycloak.getUserInfoMiddleware(),
+  asyncMiddleware(async (req, res) => {
+    await validate(ParticipantQuerySchema, req.query);
+    const user = req.hcapUserInfo;
+    const id = req.query;
+    const result = await getParticipantByID(id);
+    logger.info({
+      action: 'participant_get',
+      performed_by: {
+        username: user.username,
+        id: user.id,
+      },
+      on: {
+        id,
+      },
+    });
+    return res.json(result);
+  }));
+
+// Update participant data
+app.patch(`${apiBaseUrl}/participant`,
+  keycloak.allowRolesMiddleware('ministry_of_health'),
+  keycloak.getUserInfoMiddleware(),
+  asyncMiddleware(async (req, res) => {
+    console.log('req.body');
+    console.log(req.body);
+    await validate(ParticipantEditSchema, req.body);
+    const user = req.hcapUserInfo;
+    const result = await updateParticipant(req.body);
+    logger.info({
+      action: 'participant_get',
```

Comment:
Think this should reflect the correct action being taken.
```suggestion
      action: 'participant_updated',
```